.gitignore

```diff
@@ -12,3 +12,4 @@ security/*.req
 security/*.pem
 security/*.der
 security/*.p12
+generated_ssl_files
```

Comment:
Was that included by accident?

Reply:
the "generated_ssl_files" folder is an artifact when SSL is enabled, used for storing keys and certs

Reply:
I agree that it is useful to ignore that directory, I was just wondering why that addition had been included in this pull request as it seems unrelated :)